code/Test_definitions/qod-provisioning-createProvisioning.feature

```diff
@@ -10,11 +10,11 @@ Feature: CAMARA QoD Provisioning API, v0.2.0-rc.1 - Operation createProvisioning
     # * A device object applicable for QoD provisioning service
     # * A device object identifying a device commercialized by the implementation for which the service is not applicable, if any
     #
-    # References to OAS spec schemas refer to schemas specified in qod-provisioning.yaml, version 0.2.0-rc.1
+    # References to OAS spec schemas refer to schemas specified in qod-provisioning.yaml, version wip

```

Comment:
Let's get rid of this duplication, it's enough to have it in the title and resource
```suggestion
    # References to OAS spec schemas refer to schemas specified in qod-provisioning.yaml

```